.env.template

```diff
@@ -31,3 +31,6 @@ FRA_MAIL_PASSWORD=bar
 
 #JWT
 TOKEN_SECRET=195...95
+
+# Basic http authorization token. Created from Sepal username and password.
+SEPAL_TOKEN="xyz1234"
```

Comment:
is this a static token ever?

Reply:
Yes, it's just base64 encoding of `username:password`. Of course we can also put Sepal username and password in .env and base64 encode within code. That probably makes it clearer.

Reply:
please, share it separately ;)